frontend/src/ui/components/atoms/event-card/EventCard.tsx

```diff
@@ -26,7 +26,11 @@ function EventCard({
   return (
     <div className="eventCard" data-testid="event-card">
       <div className="eventCard__dateDetails">
-        <div className="date" style={{ backgroundColor: colour }}>
+        <div
+          className="date"
+          data-testid="event-card-date"
```

Comment:
You already are checking the day and month. data test id its not necessary

Reply:
We can remove this data-testid because we are checking the container of the date but not the day or the month :)

Reply:
You mean can't right?

You have already check the day and the month here.

expect(screen.getByText('01')); -> Day
expect(screen.getByText('Jan')); -> Month

Reply:
So yo dont need any data test id

Reply:
Yes, I meant "can't"😅. This test is about checking the background color of the container, not the date.